apps/webpack.js

```diff
@@ -111,6 +111,12 @@ var baseConfig = {
         'javalab',
         'locale-do-not-import.js'
       ),
+      '@cdo/music/locale': path.resolve(
```

Comment:
The only relevant changes to this file are here and on line 361 (adding the @cdo/music/local path alias) - the rest seem to be linter changes.

Reply:
No big deal, but one idea is to do separate PRs with linting improvements, so that functional PRs are a bit easier to read for future visitors.

Reply:
Yep for sure, I don't think this file actually needs to be linted (as it's not in apps/src), but just got formatted along the way so I included it. I can revert the formatting if that makes it easier to parse!

Reply:
Oh, one solution is to lint in a separate PR and merge that first.